src/io/flutter/view/FlutterView.java

```diff
@@ -335,32 +334,31 @@ private void addDisabledTab(String displayName,
   }
 
   private void addPerformancePlaceholderTab(JBRunnerTabs runnerTabs,
+                                            FlutterApp app,
                                             boolean selectedTab) {
-    final JPanel labelRow = new JPanel(new HorizontalLayout());
-    labelRow.add(new JLabel("Open the "));
-
-    final LinkLabel<String> linkLabel = new LinkLabel<>("Flutter Performance window", null);
+    final LinkLabel<String> linkLabel = new LinkLabel<>("See Flutter Performance window", null);
     linkLabel.setListener(new LinkListener<String>() {
       @Override
       public void linkSelected(LinkLabel aSource, String aLinkData) {
-        showFlutterPerformanceWindow();
+        showFlutterPerformanceWindow(app);
       }
     }, null);
-    labelRow.add(linkLabel);
-
-    labelRow.add(new JLabel(" for performance information"));
+    linkLabel.setBorder(JBUI.Borders.empty(3, 10));
+    linkLabel.setHorizontalAlignment(SwingConstants.CENTER);
+    if (FlutterUtils.isAndroidStudio()) {
+      // Remove underline for Android Studio to avoid LinkLabel bug where underline is left aligned even though text is center aligned.
```

Comment:
nit: add TODO to remove this check once AndroidStudio is based on IntelliJ 182 instead of 181

Reply:
Done.